src/app/clusters/scenes/SceneTableImpl.h

```diff
@@ -183,23 +183,19 @@ class DefaultSceneTableImpl : public SceneTable
     CHIP_ERROR RemoveSceneTableEntryAtPosition(FabricIndex fabric_index, SceneIndex scened_idx) override;
 
     // SceneHandlers
-    CHIP_ERROR RegisterHandler(SceneHandler * handler);
-    CHIP_ERROR UnregisterHandler(uint8_t position);
+    CHIP_ERROR RegisterHandler(SceneHandler * handler) override;
+    CHIP_ERROR UnregisterHandler(uint8_t position) override;
```

Comment:
Suggest unregister by `SceneHandler *` instead. A simpler approach would be to omit `UnregisterHandler()` for now (low likelihood of individual unregistration) and rather start with having a `UnregisterAllHandlers`.

Reply:
Suggestion was applied in latest commit.